src/app/bav/steps.js

```diff
@@ -26,11 +26,24 @@ module.exports = {
   },
   [`${APP.PATHS.CONFIRM_DETAILS}`]: {
     controller: confirmDetails,
-    next: APP.PATHS.DONE,
-  },
-  [`${APP.PATHS.CONFIRM_DETAILS}`]: {
-    controller: confirmDetails,
-    next: APP.PATHS.DONE,
+    fields: ["retryCount"],
+    next: [
+      {
+        field: "retryCount",
+        value: 0,
```

Comment:
If there is a full match that backend will not send a `retryCount` so do we need to add some logic to the `req.sessionModel.set` to set 0 in that case?

Reply:
I can add this in but it is outside the ticket scope, or add it as part of a subsequent ticket that will deal with a successful attempt?

Reply:
Does the existing functionality that we had before this ticket still work? I'm concerned that we are breaking something by doing this

Reply:
Previous functionality was just another placeholder that looked like this:
[`${APP.PATHS.CONFIRM_DETAILS}`]: {
    controller: confirmDetails,
    next: APP.PATHS.DONE,
}
DONE just representing the end of the journey